stable/minio/values.yaml

```diff
@@ -69,7 +69,7 @@ persistence:
 
 service:
   type: ClusterIP
-  clusterIP: None
+  clusterIP:
```

Comment:
Can you set as `clusterIP: ~`   ( ~ is the [canonical](http://yaml.org/type/null.html) way to specify a null resource and is less confusing than an empty value ).

Reply:
Done @paulczar . Thanks for the review